pandas/io/gbq.py

```diff
@@ -106,13 +107,14 @@ class TableCreationError(PandasError, ValueError):
     """
     pass
 
-
 class GbqConnector(object):
 
-    def __init__(self, project_id, reauth=False):
+    def __init__(self, project_id, reauth=False, private_key_json_path=None):
         self.test_google_api_imports()
```

Comment:
update the doc-string.

let's call this ``private_key`` (which can also be a string-like yes?)